### PR TITLE
quiz models introduced

### PR DESCRIPTION
**Pull request by nyckoo**
there's still need to improve here and there :x

### CODE REVIEW

File: apps/quiz/models.py [hunk 1]
```diff
@@ -1,5 +1,43 @@
-from database import db
+from database.database import db
 
 
 class Quiz(db.Model):
-    pass
+
+    __tablename__ = 'quiz'
```
Comment (mtyton) [inline apps/quiz/models.py]: from what I know tablename does not have to be explicitly defined

File: apps/quiz/models.py [hunk 2]
```diff
@@ -1,5 +1,43 @@
...
+
+    def __init__(self, category, difficulty_level, questions):
+        self.category = category
+        self.difficulty_level = difficulty_level
+
+
+class QuizCategory(db.Model):
+
+    __tablename__ = 'quiz_category'
+
```
Comment (mtyton) [inline apps/quiz/models.py:20]: There should also be a category_name

File: apps/quiz/models.py [hunk 3]
```diff
@@ -1,5 +1,43 @@
...
+    id = db.Column(db.Integer, primary_key=True)
+
+
+class Question(db.Model):
+
+    __tablename__ = 'question'
+
+    id = db.Column(db.Integer, primary_key=True)
+    text = db.Column(db.String(500), db.ForeignKey('answer.text'))
+    question_type = db.Column(db.String(150))
```
Comment (mtyton) [inline apps/quiz/models.py]: As I said, I'll handle those fields by creating enums

File: apps/quiz/models.py [hunk 4]
```diff
@@ -1,5 +1,43 @@
...
+
+    id = db.Column(db.Integer, primary_key=True)
+
+
+class Question(db.Model):
+
+    __tablename__ = 'question'
+
+    id = db.Column(db.Integer, primary_key=True)
+    text = db.Column(db.String(500), db.ForeignKey('answer.text'))
```
Comment (mtyton) [inline apps/quiz/models.py]: This definietly should not be like this, using others table not primary key as foreign key is quite unwise, will explain this today's evening

File: apps/quiz/models.py [hunk 5]
```diff
@@ -1,5 +1,43 @@
...
+
+    def __init__(self, category, difficulty_level):
+        self.category = category
+        self.difficulty_level = difficulty_level
+
+
+class Answer(db.Model):
+
+    __tablename__ = 'answer'
+
```
Comment (mtyton) [inline apps/quiz/models.py:60]: missing primary key

**Review by mtyton — COMMENTED** [review]
We will discuss those comments today's evening

**Review by nyckoo — COMMENTED** [review]
there's only enum left, didn't know if "choices" are suitable here soo i'll be glad to see how you code it :>

**Pull request merged**